lib/emscripten/src/lib.rs

```diff
@@ -1,15 +1,17 @@
 #[macro_use]
 extern crate wasmer_runtime_core;
+extern crate wasmer_runtime;
```

Comment:
@lachlansneff Should this depend on `wasmer_runtime`?

Reply:
during `cargo build --all` I see error[E0463]: can't find crate for `wasmer_runtime`
```
 --> lib/emscripten/src/lib.rs:3:1
  |
3 | extern crate wasmer_runtime;
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ can't find crate
```

Reply:
I'll just use `cargo build --all --exclude wasmer-emscripten` for now.